addons/account_edi_ubl_cii/tests/test_ubl_cii.py

```diff
@@ -377,15 +377,24 @@ def test_import_discount(self):
         invoice = self.env['account.move'].create({
             'partner_id': self.partner_a.id,
             'move_type': 'out_invoice',
-            'invoice_line_ids': [Command.create({
+            'invoice_line_ids': [
+                Command.create({
                 'product_id': self.product_a.id,
                 'quantity': 3,
                 'price_unit': 11.34,
-            })],
+                }),
+                Command.create({
+                'product_id': self.product_a.id,
+                'quantity': 1.65,
+                'price_unit': 29.9,
+                }),
+            ],
         })
         xml_attachment = self.env['ir.attachment'].create({
             'raw': self.env['account.edi.xml.cii']._export_invoice(invoice)[0],
             'name': 'test_invoice.xml',
         })
         imported_invoice = self.import_attachment(xml_attachment, self.company_data["default_journal_sale"])
-        self.assertFalse(imported_invoice.invoice_line_ids.discount)  # if slight rounding error won't be falsy
+        # if slight rounding error won't be falsy
+        for line in imported_invoice.invoice_line_ids:
+            self.assertFalse(line.discount, "Discount should be falsy for all imported invoice lines")
```

Comment:
There's no need to repeat what the line does, but you can underline the problem instead.
```suggestion
            self.assertFalse(line.discount, "Any discount on the imported lines signals a rounding error in the total")
```